sdk/devcenter/Azure.Developer.DevCenter/tests/DevCenterClientTestEnvironment.cs

```diff
@@ -20,7 +20,7 @@ public class DevCenterClientTestEnvironment : TestEnvironment
         private string _scope => GetRecordedVariable("DEFAULT_DEVCENTER_SCOPE");
         private string _testUserSecret => GetRecordedVariable("DEFAULT_TEST_USER_SECRET", options => options.IsSecret());
         private string _testUserName => GetRecordedVariable("DEFAULT_TEST_USER_NAME");
-        public Uri Endpoint => new(GetRecordedVariable("DEFAULT_DEVCENTER_ENDPOINT"));
+        public Uri Endpoint => new(GetRecordedVariable("DEVCENTER_ENDPOINT"));
```

Comment:
[nit]Why the change? You have been using the DEFAULT_ naming convention for other variables.

Reply:
Good catch :)